cbor-file-magic.mkd

```diff
@@ -63,11 +63,14 @@ is friendly to common on-disk recognition systems such as the Unix file(1) comma
 # Introduction
 
 Since very early in computing, operating systems have sought ways to mark which files could be processed by which programs.
+At one point, files were all record orientated and the operating system enforced this.
```

Comment:
Well, this is a somewhat categorical statement that is not true at that level.  There were "Access methods", some of which were record-based, some of which weren't.

Reply:
my intent here is to say that, at the beginning of time, the OS knew how to process every file, and then when we went to everything is a stream of bytes, we lost this.  Maybe this isn't worth saying at all?